python-sdk/nuscenes/eval/lidarseg/evaluate.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+from tqdm import tqdm
+import numpy as np
+
+from nuscenes import NuScenes
+
+from utils import LidarsegChallengeAdaptor
+
+
+class LidarSegEval:
+    def __init__(self,
+                 nusc: NuScenes,
+                 results_folder: str,
+                 ignore_idx: int = 0):
+        """
+        """
+
+        """
+        # Check there are ground truth annotations.
+        self.gt_samples = nusc.sample # check if test set has samples?? Might have to check data itself.
+        assert len(self.samples) > 0, 'Error: There are no ground truth ______'
```

Comment:
??

Reply:
These are messy WIP comments for myself; pls ignore, haha